process/mockoon-usage.md

```diff
@@ -0,0 +1,317 @@
+   - [Starting Mockoon Server](#starting-mockoon-server)
+   - [Using Postman to Call API](#using-postman-to-call-api)
+   - [Copying and Adding Route Configuration](#copying-and-adding-route-configuration)
+   - [Toggling and Deleting Routes](#toggling-and-deleting-routes)
+5. [Advanced Usage](#5-advanced-usage)
+   - [Using Mockoon Templating](#using-mockoon-templating)
+   - [Using Data Buckets](#using-data-buckets)
+   - [Adding and Configuring CRUD Routes](#adding-and-configuring-crud-routes)
+   - [Using Proxy Mode](#using-proxy-mode)
+   - [Requests Logging and Recording](#requests-logging-and-recording)
+   - [Import/Export Swagger Documentation](#importexport-swagger-documentation)
+   - [Adding Global Routes with Rules](#adding-global-routes-with-rules)
+   - [Creating and Using Callbacks](#creating-and-using-callbacks)
+6. [Further References](#6-further-references)
+
+## 1. Introduction
+
+### The Problem
+
+There is currently accelerating hindrance in frontend development due to API dependencies across different projects. If the API development is not delivered on time, it can cause delays in development and integration in the frontend as well.
+
+### Current Pipeline
+
+![Current Pipeline](assets/mockoon-without-pipeline.png)
+
+This pipeline requires the backend developer to always be one step ahead of the frontend developer in development of a particular module.
+
+### The Solution
+
+There are two possible ways to avoid delays due to the above pipeline:
+
+- Provide API details to the frontend before development. We can try to use AI tools to do this, but it has been found that these are not reliable. This will also require human intervention for testing the APIs, for example, using different responses.
+- Use a mock API server. This will mock the API responses using static or random responses as configured.
+
+[Back to top](#mockoon-usage)
+
+## 2. Why Mockoon?
+
+[Mockoon](https://mockoon.com) is a free and open-source application that is cross-platform and provides an easy and quick way to run mock APIs locally. There is no mandatory login or account requirement for using this.
+
+### Steps
+
+- Backend developer adds Mockoon configuration for endpoints
+- Frontend developer imports API configuration into his local Mockoon server
+
+### Mockoon Pipeline
+
+![Mockoon Pipeline](assets/mockoon-with-pipeline.png)
+
+In this pipeline, the frontend developer does not have to wait for the complete development and deployment of the API to start working on the component and integration in the frontend.
+
+[Back to top](#mockoon-usage)
+
+## 3. Setup
+
+Go to the [Mockoon website](https://mockoon.com) and head to the [download page](https://mockoon.com/download/#download-section). Download and install the required package as per your OS.
+
+Take the tour to gain an understanding of the various UI elements and features.
+
+[Back to top](#mockoon-usage)
+
+## 4. Usage
```

Comment:
In the usage.
- You need to mention what are expected steps from Backend Guys.
-  And what are the expected actions and features to be used by frontend guys(like playing with latency, responses Sequential/Random Response modes) 

Specify importance of  topics like adding different responses, latency(global and local), Sequential/Random Response Modes and Proxy Server. And next would be templating.

Keep others topics separate so Importance of usage will be clearer. Also specify about certain features that they wont be used(Example CRUD feature is useless for our process, so we wont want people spending time on it, but just know of the feature).

Reply:
Addressed in https://github.com/OsmosysSoftware/dev-standards/pull/49/commits/78438cdadc6d635ecc757d32148cb97f91835fd7